eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/myworkorders/MyWorkOrders.java

```diff
@@ -89,7 +89,7 @@ public List<MyWorkOrder> getObjectEvents(String equipmentCode, String equipmentT
         workOrdersGridRequest.addParam("parameter.object", equipmentCode);
 
         // use star organization to get all work orders, independently of organization
-        workOrdersGridRequest.addParam("parameter.objorganization", "*");
+        workOrdersGridRequest.addParam("parameter.objorganization", authenticationTools.getR5InforContext().getOrganizationCode() != null ? authenticationTools.getR5InforContext().getOrganizationCode() : "*");
```

Comment:
authenticationTools.getR5InforContext() is called more than once in this method, extract as a variable

Reply:
Also, the organization should be defined in the context anyways (for Open and CERN), so no need for the "*" default case